src/index.js

```diff
@@ -0,0 +1,213 @@
+  showCity.innerHTML = inputCity;
+  currentCity = inputCity;
+};
+
+const resetCity = () => {
+  let cityInput = document.getElementById('cityName');
+  cityInput.value = 'Atlanta';
+  updateCity();
+};
+
+// CONDITIONS
+
+let selectConditions = document.querySelectorAll('input[type="radio"]');
+
+let condResult = document.getElementById('cond_result');
+condResult.innerHTML = '✨';
+
+const updateCondition = () => {
+  try {
+    let selected = document.querySelector(
+      "input[name='Conditions']:checked"
+    ).value;
+    condResult.innerHTML = selected;
+  } catch {}
+};
+
+// TEMP DEPENDANTS
+
+let defaultTemp = 'LOADING...';
+let currentTemp = defaultTemp;
+
+const HOW_YA_FEELIN = ['🥶', '😮‍💨', '😄', '😎', '😅', '🥵', '🤔'];
+const HILLS_ARE_ALIVE = ['🧊', '🌲', '🌷', '🌻', '🌴', '🔥', '🌎'];
+const REPO_RAINBOW = [
+  '#993BDB',
+  '#2B52FF',
+  '#0BD769',
+  '#FFE146',
+  '#FF7912',
+  '#FF0000',
+];
+
+let showField = document.getElementById('temp_field');
+let showYou = document.getElementById('you_are_here');
+
+const updateTempies = (adjustedTemp) => {
+  let tempSetting = 6;
+
+  if (adjustedTemp > 99) {
+    tempSetting = 5;
+  } else if (adjustedTemp > 79) {
+    tempSetting = 4;
+  } else if (adjustedTemp > 65) {
+    tempSetting = 3;
+  } else if (adjustedTemp > 39) {
+    tempSetting = 2;
+  } else if (adjustedTemp > 14) {
+    tempSetting = 1;
+  } else if (adjustedTemp <= 14) {
+    tempSetting = 0;
+  }
+
+  currentTemp = adjustedTemp;
+
+  showField.innerHTML = HILLS_ARE_ALIVE[tempSetting];
+  showYou.innerHTML = HOW_YA_FEELIN[tempSetting];
+  shownTemp.innerHTML = adjustedTemp;
+  document.getElementById('temp_result').style.color =
+    REPO_RAINBOW[tempSetting];
+};
+
+// TEMPERATURE
+
+let shownTemp = document.getElementById('temp_result');
+
+shownTemp.innerHTML = currentTemp;
+
+const handleAdd = () => {
+  currentTemp++;
+  shownTemp.innerHTML = currentTemp;
+
+  updateTempies(currentTemp);
+};
+
+const handleSub = () => {
+  currentTemp--;
+  shownTemp.innerHTML = currentTemp;
+
+  updateTempies(currentTemp);
+};
+
+// API CALLS
+
+const findWeather = (query) => {
+  let locationData, weatherData;
+  let modifiedLocData, modifiedWeathData;
+
+  axios
+    .get('http://127.0.0.1:5000/location', {
+      params: {
+        q: currentCity,
+        format: 'json',
+      },
+    })
+    .then((response) => {
+      locationData = response.data;
+
+      const modifyLocation = (locationData) => {
+        let latlon = [];
+        let focusLocation = locationData[0];
+
+        let focusLat = Number(focusLocation['lat']);
+        let adjustLat = focusLat.toFixed(2);
+        latlon.push(adjustLat);
+
+        let focusLon = Number(focusLocation['lon']);
+        let adjustLon = focusLon.toFixed(2);
+        latlon.push(adjustLon);
+
+        return latlon;
+      };
+
+      modifiedLocData = modifyLocation(locationData);
+
+      axios
+        .get('http://127.0.0.1:5000/weather', {
+          params: {
```

Comment:
Consider breaking this function up a bit. It's taking on a lot responsibilty. Whenever we see a function that is more than 10 lines long we should consider seprating concerns. This is completely preferential but just a note. More on this here:

https://dmitripavlutin.com/the-art-of-writing-small-and-plain-functions/

https://medium.com/@janakachathuranga/coding-best-practices-javascript-write-small-functions-7d2567bd6328